src/main/java/module/playerOverview/PlayerDetailsPanel.java

```diff
@@ -640,7 +571,6 @@ private void findComparisonPlayer() {
      * initialize all fields
      */
     private void initComponents() {
```

Comment:
the layout doesn't look ready yet. 
I would suggest displaying the avatar in the top left cells (maybe one column wide and several or all rows deep?)

Reply:
yes layout not fully finished (at least when you look at it 3 days ago) ... yesterday was much closer to what it will look in the end .. but still couple of things to be done ....